### ISSUE TITLE
Confirmation emails to both myself and respondent are not being sent

### ISSUE THREAD

**Issue by TMCmssm** [sysnetlab/GoogleFormsNotifications#3]
This worked a few weeks ago.  Has it been discontinued?

**Comment by sysnetlab** [comment]
No it has not been discontinued. Did you happen to run out of you email quota with Google? If you run out of quota, the addon cannot send out any emails from your account.

**Comment by TMCmssm** [comment]
I don’t think so.  We just started using Forms and the extender a few weeks ago and up until a couple days ago we’d only tested it (maybe like 20 emails) to get it like we wanted.

We didn’t send the email directly from within Google Forms but instead used a link and sent from our own email.

**Comment by TMCmssm** [comment]
Just looked into quotas.  Not close to running out.  If it worked for one form and then we copy that form and select the same extender options, is there a glitch that doesn’t allow it to continue working?  That’s the only thing I can think of because on the original test form we used it worked like a charm.
